packages/cli/src/menus/main.ts

```diff
@@ -122,3 +140,23 @@ export async function simpleSend(state: State): Promise<State> {
   await transactionDialog(tx.toCbor().toString(), false, state);
   return state;
 }
+
+export async function registerStakeKey(state: State): Promise<State> {
+  await printHeader(state);
+  console.log("\t==== Register Stake Key ====\n");
+  const stakeHash = await input({
+    message: "Enter stake key hash (in hex)",
+  });
+  const tx = await state
+    .sdk!.blaze()
```

Comment:
If we are forcing the value of `sdk` here, can we make it non-optional in the `state` typings to avoid it?